### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AnilNITT**

### CODE REVIEW

**Comment by AnilPYT** [comment]
@AnilNITT ok done

File: README.md [hunk 1]
```diff
@@ -82,7 +82,7 @@
   <img alt="Keras" src="https://img.shields.io/badge/Keras-%23D00000.svg?style=for-the-badge&logo=Keras&logoColor=white"/>
   <img alt="SciPy" src="https://img.shields.io/badge/SciPy-%230C55A5.svg?style=for-the-badge&logo=scipy&logoColor=%white"/>
   <br>
-</div>
+</div>
```
Comment (AnilPYT) [inline README.md:85]: yeah done

**Review by AnilPYT — APPROVED** [review]
@AnilNITT

**Comment by AnilNITT** [comment]
Co-authored-by: @AnilNITT

**Pull request merged**